docs/pages/choose-an-edition/teleport-team.mdx

```diff
@@ -39,194 +41,250 @@ web & tsh--Client traffic-->proxy
+
+    ```code
+    > Profile URL:      https://tenant-url.teleport.sh:443
+    Logged in as:       admin@teleport.example.com
+    Cluster:            tenant-url.teleport.sh
+    Roles:              access, auditor, editor
+    Logins:             root
+    Kubernetes:         enabled
+    Valid until:        2023-07-08 01:35:20 -0700 PDT [valid for 12h0m0s]
+    Extensions:         login-ip, permit-agent-forwarding, permit-port-forwarding, permit-pty, private-key-policy
+
+1. List the servers your Teleport user can access. 
+
+    ```code
+    $ tsh ls
+    ```
+
+    You should see the name of the container you just registered. For example:
+    
+    ```code
+    Node Name    Address    Labels
+    ------------ ---------- ----------------------------------------------------------------------------------------
+    000000000000 ⟵ Tunnel   hostname=000000000000,teleport.internal/resource-id=000000000000
+    ```
+
+1. Access your server as the `root` user:
+
+    ```code
+    $ tsh ssh root@<Var name="node-name" />
+    ```
 
 ## Next steps
 
-We have shown you how to use Teleport Team to quickly protect your
-infrastructure by registering a server with your Teleport Team cluster. From
-here, you can explore more of the documentation to see how to set up secure
-access for your infrastructure.
+This guide introduced how you can use Teleport Team to protect your
+infrastructure by demonstrating how to register a server with your Teleport Team 
+tenant. 
+
+You can provide secure access to more of your infrastructure through 
+Teleport Team by deploying one or more Teleport **agents** and configuring
+role-based access control for users. Agents proxy traffic to all of 
+your infrastructure resources—including servers, databases, Kubernetes
+clusters, cloud provider APIs, and Windows desktops. Role-based access control
+ensures that only authorized users are allowed access to those resources.
 
-To enroll your infrastructure with Teleport, you deploy one or more Teleport
-**agents**, which proxy traffic to resources like servers, databases, Kubernetes
-clusters, cloud provider APIs, and Windows desktops.
+For more information about deploying agents, see the following topics:
+- [Run Teleport Agents](../agents/introduction/)
+- [Deploy Teleport Agents withTerraform](../agents/deploy-agents-terraform.mdx)
+- [Join Services to your Teleport Cluster](../agents/join-services-to-your-cluster/)
 
-You can use Terraform to declare a pool of Teleport agents and configure them to
-proxy your infrastructure. Read [Deploy Teleport Agents with
-Terraform](../agents/deploy-agents-terraform.mdx).
+For information about configuring roles, see the following topics:
+- [Getting Started with Access Controls](../access-controls/getting-started/)
+- [Cluster Access and RBAC](../access-controls/guides/)
+- [Access Controls for Servers](../server-access/rbac/#role-configuration)
```

Comment:
We should probably get feedback from @xinding33 on this part of the page. We wanted to include a learning track that would emphasize infrastructure as code and guide new Team/OSS users to a page about deploying agents with Terraform. I want to make sure this has the right level of emphasis here.